www/js/diary/list.js

```diff
@@ -199,6 +199,9 @@ angular.module('emission.main.diary.list',['ui-leaflet',
         tripgj.display_time = $scope.getFormattedTimeRange(tripgj.data.properties.start_ts,
                                 tripgj.data.properties.end_ts);
         tripgj.isDraft = $scope.isDraft(tripgj);
+        // rciti start
+        tripgj.isAnalyzed = $scope.isAnalyzed(tripgj);
+        // rciti end
```

Comment:
I would prefer using `isDraft` (if possible) to avoid rciti specific code to the maximum extent

Reply:
migrated to `isDraft`.